src/server/cdn-controller.js

```diff
@@ -30,16 +31,17 @@ class CdnController {
     });
 
     router.post('/api/v1/cdn/objects', [needsPermission(permissions.CREATE_FILE), multipartParser.array('files'), validateBody(postObjectsBodySchema)], async (req, res) => {
+      const utcNowString = moment.utc().toISOString();
```

Comment:
But but but, I thought we wanted to move away from moment :(

Reply:
Sure, I didn't want to do that in this PR though :thinking:

Reply:
I was half kidding :)

Reply:
Puh :-)